src/System.Windows.Forms/src/System/Windows/Forms/ListViewLabelEditAccessibleObject.cs

```diff
@@ -0,0 +1,101 @@
+                UiaCore.UIA.BoundingRectanglePropertyId => Bounds,
+                UiaCore.UIA.ProcessIdPropertyId => Environment.ProcessId,
+                UiaCore.UIA.ControlTypePropertyId => UiaCore.UIA.EditControlTypeId,
+                UiaCore.UIA.NamePropertyId => Name,
+                UiaCore.UIA.AccessKeyPropertyId => string.Empty,
+                UiaCore.UIA.HasKeyboardFocusPropertyId => true,
+                UiaCore.UIA.IsKeyboardFocusablePropertyId => (State & AccessibleStates.Focusable) == AccessibleStates.Focusable,
+                UiaCore.UIA.IsEnabledPropertyId => Enabled,
+                UiaCore.UIA.AutomationIdPropertyId => LIST_VIEW_LABEL_EDIT_AUTOMATION_ID,
+                UiaCore.UIA.HelpTextPropertyId => Help ?? string.Empty,
+                UiaCore.UIA.IsContentElementPropertyId => true,
+                UiaCore.UIA.IsPasswordPropertyId => false,
+                UiaCore.UIA.NativeWindowHandlePropertyId => _handle,
+                UiaCore.UIA.IsOffscreenPropertyId => false,
+                UiaCore.UIA.IsTextPatternAvailablePropertyId => IsPatternSupported(UiaCore.UIA.TextPatternId),
+                UiaCore.UIA.IsTextPattern2AvailablePropertyId => IsPatternSupported(UiaCore.UIA.TextPattern2Id),
+                UiaCore.UIA.IsValuePatternAvailablePropertyId => IsPatternSupported(UiaCore.UIA.ValuePatternId),
+                UiaCore.UIA.IsLegacyIAccessiblePatternAvailablePropertyId => IsPatternSupported(UiaCore.UIA.LegacyIAccessiblePatternId),
+                _ => base.GetPropertyValue(propertyID),
+            };
+
+        internal override UiaCore.IRawElementProviderFragment? FragmentNavigate(UiaCore.NavigateDirection direction)
+        {
+            AccessibleObject parent = _owner.AccessibilityObject;
+
+            return direction switch
+            {
+                UiaCore.NavigateDirection.Parent => parent,
+                UiaCore.NavigateDirection.NextSibling => parent.GetChildIndex(this) is int childId and >= 0 ? parent.GetChild(childId + 1) : null,
+                UiaCore.NavigateDirection.PreviousSibling => parent.GetChildIndex(this) is int childId and >= 0 ? parent.GetChild(childId - 1) : null,
+                UiaCore.NavigateDirection.FirstChild or UiaCore.NavigateDirection.LastChild => null,
+                _ => base.FragmentNavigate(direction),
+            };
+        }
+
+        internal override bool IsPatternSupported(UiaCore.UIA patternId) => patternId switch
+        {
+            UiaCore.UIA.TextPatternId => true,
+            UiaCore.UIA.TextPattern2Id => true,
+            UiaCore.UIA.ValuePatternId => true,
+            UiaCore.UIA.LegacyIAccessiblePatternId => true,
+            _ => base.IsPatternSupported(patternId),
+        };
+
+        internal override int[] RuntimeId => _runtimeId ??= new int[] { RuntimeIDFirstItem, PARAM.ToInt(_handle) };
+
+        internal override UiaCore.IRawElementProviderFragmentRoot? FragmentRoot => _owner.AccessibilityObject;
```

Comment:
Can AccessibilityObject value be null?
```suggestion
        internal override UiaCore.IRawElementProviderFragmentRoot FragmentRoot => _owner.AccessibilityObject;
```